api/candlepin-api-spec.yaml

```diff
@@ -30,6 +30,336 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+  /activation_keys:
+    get:
+      description: Retrieves a list of activation keys
+      tags:
+        - Activation Key
+      operationId: findActivationKey
+      x-java-response:
+        type: Iterable
+        isContainer: true
+      security: []
+      responses:
+        200:
+          description: List of activation keys
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ActivationKeyDTO'
+        default:
+          $ref: '#/components/responses/default'
+
+  /activation_keys/{activation_key_id}:
+    get:
+      description: Retrieves a single activation key
+      tags:
+        - Activation Key
+      operationId: getActivationKey
+      security: []
+      parameters:
+        - name: activation_key_id
+          in: path
+          description: The Id of activation key to retrieve
```

Comment:
ID should be capitalized when used in a description or display message.

Reply:
Done.